### PR TITLE
Add docs to all public methods and properties of `RubberbandModel`

### PR DESCRIPTION
**Pull request by suricactus**
The removed method might raise questions, feel free to revert if I misunderstood it's usage and purpose.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for e806d3cfcecfed80667e99394731b87696162595: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4814-c37c2175a94650db321973bb702f96d906e7392b-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4814-c37c2175a94650db321973bb702f96d906e7392b-arm-neon-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4814-c37c2175a94650db321973bb702f96d906e7392b-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4814-c37c2175a94650db321973bb702f96d906e7392b-x86-android.apk)
- [**all access arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4814-c37c2175a94650db321973bb702f96d906e7392b-arm64-android.apk)
- [all access arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4814-c37c2175a94650db321973bb702f96d906e7392b-arm-neon-android.apk)
- [all access x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4814-c37c2175a94650db321973bb702f96d906e7392b-x64-android.apk)
- [all access x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4814-c37c2175a94650db321973bb702f96d906e7392b-x86-android.apk)
</details>

File: src/core/rubberbandmodel.h [hunk 1]
```diff
@@ -61,82 +71,113 @@ class QFIELD_CORE_EXPORT RubberbandModel : public QObject
...
 
+    //! \copydoc vertexCount
     int vertexCount() const;
 
-    bool isEmpty() const;
-
+    //! \copydoc vertices
     QVector<QgsPoint> vertices() const;
 
-    QVector<QgsPoint> flatVertices( bool skipCurrentPoint = false ) const;
```
[the remark this answers is not in the document]
Comment (suricactus) [reply src/core/rubberbandmodel.h:70]: I didn't check historically when it was used, but not anymore with the parameter sets to true.

**Review by nirvn — APPROVED** [review]

File: src/core/rubberbandmodel.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (suricactus) [reply src/core/rubberbandmodel.h:70]: Can we eat least rename the function? It does not make sense at all.

**Comment by suricactus** [comment]
@nirvn all green, ready for merge?

**Comment by nirvn** [comment]
@suricactus , yes, let's merge.

**Pull request merged**